### DISCUSSION TITLE
字典自动索引前缀

### DISCUSSION THREAD

**Discussion by tristone13th** [rime/home#1054] [Q&A]
如果我想在我的字典或者 custom_phrase.txt 里添加一个映射，比如从 `abcd` 到 `abcd` 我可以：
```
abcd    abcd
```
如果我想在我输入到 b 时就触发这个映射，我可以改成：
```
abcd    ab
abcd    abc
abcd    abcd
```
请问以上的写法有没有一种比较简便的方式呢，谢谢！

**Comment by iDvel** [comment]
GitHub Copilot 有点帮助。。。
![2022-04-29-000608](https://user-images.githubusercontent.com/14658234/165977906-6cb7bd0c-2924-4b33-bda5-f87a32391899.gif)

**Reply by tristone13th** [reply]
copillot 是不是只支持固定的编辑器呀，如果想通用一些比如在浏览器里是不是就不可以

**Reply by iDvel** [reply]
用 VSCode 编辑呗

**Reply by tristone13th** [reply]
搜索的时候可能需要用到这个功能
